src/styles/variables.scss

```diff
@@ -0,0 +1,2 @@
+$speed-short: 5s;
```

Comment:
It is a good practice to add a brief comment explaining the purpose of the variables